tests/test_config_generation.py

```diff
@@ -60,3 +61,6 @@ def test_k8s_cronjobs_beta_api_version_configs(self):
 
     def test_multiple_deployments_configs(self):
         _test_configs('multiple_deployments')
+
+    def test_registry_configs(self):
+        _test_configs('docker_registry', 'default-registry')
```

Comment:
Why is this not following the same convention as all the other tests, named `test_<folder_name>_configs`?

Reply:
https://github.com/EDITD/kubetools/pull/141/commits/b69af2413f6bf45f793c6462ce127b2478927dd2